worlds/inscryption/test/TestGoal.py

```diff
@@ -0,0 +1,80 @@
+from . import InscryptionTestBase
+
+
+class GoalTestOrdered(InscryptionTestBase):
+    options = {
+        "goal": 0,
+    }
+
+    def test_beatable(self) -> None:
+        for item_name in self.required_items_all_acts:
```

Comment:
This basically adds every required item one by one and tests that the game is not beatable before collecting it. After everything is collected, it tests that the game is now beatable. The epitaph pieces are separate from the "required_items_all_acts" simply because the item name depends on the option as you can see in the other tests.

Reply:
I think these should be using AssertAccessDependency then, since you're really only testing the _entire_ set and not each item

Reply:
Should I just remove the False checks while keeping the final one? Or are these tests just not needed?

Reply:
I think a better way to do these would be to give yourself every item, assert beatability, then remove one, assert _not_ beatability, put it back and remove the next (different) one.

Reply:
Smart. Gonna give it a try.